test/hooks.test.js

```diff
@@ -74,6 +74,35 @@ test('hooks on static swagger', async t => {
   }
 })
 
+test('onSend hook on static swagger', async t => {
+  const fastify = Fastify()
+  await fastify.register(fastifySwagger, {
+    mode: 'static',
+    specification: {
+      path: './examples/example-static-specification.yaml'
+    }
+  })
+  await fastify.register(fastifySwaggerUi, {
```

Comment:
So you basically adding onSend Hook to all routes. So if you do a request against "/documentation/uiConfig" you get also the modified swagger json.

Reply:
`onSend` hook is more generic and not limited in changing `swagger` output only.
They can also mutate the `config`, etc.

`uiHook` like `onRequest` and `preHandler` also spread across all the route.
The users should be aware on the limitation and they need to check the `url` to match dedicated route.

Reply:
The unit test already shows me, that it is not intuitive to implement it correctly.

Reply:
I would argue that even with `onRequest` or `preHandler`.
People would make it wrongly.